e1000d/src/device.rs

```diff
@@ -87,11 +108,17 @@ struct Td {
     css: u8,
     special: u16,
 }
+
+// TODO: One // line here to describe what this GROUP of constants is
+// TODO: leave one line description of these using ///
 const TD_CMD_EOP: u8 = 1;
 const TD_CMD_IFCS: u8 = 1 << 1;
 const TD_CMD_RS: u8 = 1 << 3;
 const TD_DD: u8 = 1;
 
+/// The Intel8254x device and it's associated device memory
```

Comment:
I had a lot of personal questsions about where memory was living. I assume almost all of these are physical addresses, but are some of them not even in the RAM (i.e. are they on the device itself?)